### PR TITLE
[release/v2.2] Add 1.13.9, 1.14.5 and 1.15.2

### PR DESCRIPTION
**Pull request by superseb**

### CODE REVIEW

File: apis/management.cattle.io/v3/k8s_windows_default.go [hunk 1]
```diff
@@ -426,7 +431,9 @@ func initWindows() {
 		K8sVersionWindowsSystemImages[version] = images
 	}
 
-	if _, ok := K8sVersionWindowsSystemImages[DefaultK8s]; !ok {
-		panic("Default K8s version " + DefaultK8s + " is not found in k8sVersionsCurrent list")
+	if DefaultK8s < "v1.13.6" || (DefaultK8s > "v1.14.0" && DefaultK8s < "v1.14.5") || (DefaultK8s > "v1.15.1" && DefaultK8s < "v1.15.2") {
```
Comment (deniseschannon) [inline apis/management.cattle.io/v3/k8s_windows_default.go]: to get k8s `v1.15.0`,`(DefaultK8s > "v1.14.99" && DefaultK8s < "v1.15.1")`

File: apis/management.cattle.io/v3/k8s_windows_default.go [hunk 2]
```diff
@@ -417,6 +417,11 @@ func initWindows() {
 		if badVersions[version] {
 			continue
 		}
+		// we stopped building Windows images at 1.13.9 (earliest previous version is 1.13.7), 1.14.5 (earliest previous version is 1.14.3) and 1.15.2 (earliest previous version was 1.15.0)
+		// existing images are still checked
+		if (version > "v1.13.8" && version < "v1.13.99") || (version > "v1.14.4" && version < "v1.14.99") || version > "v1.15.1" {
```
Comment (deniseschannon) [inline apis/management.cattle.io/v3/k8s_windows_default.go]: `version > v1.15.0`

File: apis/management.cattle.io/v3/k8s_windows_default.go [hunk 3]
```diff
@@ -417,16 +389,23 @@ func initWindows() {
 		if badVersions[version] {
 			continue
 		}
+		// we stopped building Windows images at 1.12.10 (earliest previous version was 1.12.9), 1.13.9 (earliest previous version is 1.13.7), 1.14.5 (earliest previous version is 1.14.3) and 1.15.2 (earliest previous version was 1.15.0)
+		// existing images are still checked
+		if (version > "v1.12.10" && version < "v1.12.99") || (version > "v1.13.8" && version < "v1.13.99") || (version > "v1.14.4" && version < "v1.14.99") || version > "v1.15.1" {
```
Comment (kinarashah) [inline apis/management.cattle.io/v3/k8s_windows_default.go]: @superseb won't v1.13.7 just pass the first condition and be true here?

Comment (kinarashah) [reply apis/management.cattle.io/v3/k8s_windows_default.go]: nvm, that's what the `<` part would take care of.

**Pull request merged**